### PR TITLE
puppetdb, puppetserver: Add volumes for persistent cert storage

### PR DESCRIPTION
**Pull request by ms1111**
As discussed in #13, define some persistent volumes.

### CODE REVIEW

File: puppetdb/Dockerfile [hunk 1]
```diff
@@ -32,6 +32,11 @@ COPY logging /etc/puppetlabs/puppetdb/logging
 RUN rm -fr /etc/puppetlabs/puppetdb/conf.d
 COPY conf.d /etc/puppetlabs/puppetdb/conf.d
 
+# Persist the agent SSL certificate.
+VOLUME /etc/puppetlabs/puppet/ssl/
+# /etc/puppet/labs/puppetdb/ssl is automatically populated from here and
```
Comment (garethr) [inline puppetdb/Dockerfile]: Minor typo in the comment. `puppet/labs` rather than `puppetlabs`

**Comment by garethr** [comment]
@ms1111 thanks for this, one minor comment about a comment, if you could modify that commit I'll merge away.

**Comment by ms1111** [comment]
@garethr Thanks, comment has been fixed.

**Comment by MWilsonPuppet** [comment]
@ms1111 thanks for adding this.

**Pull request merged**